packages/react-instantsearch/src/connectors/connectConfigure.js

```diff
@@ -1,7 +1,10 @@
 import createConnector from '../core/createConnector.js';
-import {omit, isEmpty, difference, keys} from 'lodash';
+import {omit, difference, keys} from 'lodash';
+import {hasMultipleIndex, getIndex, refineValue} from '../core/indexUtils';
 
-const namespace = 'configure';
+function getId() {
```

Comment:
Indeed, then maybe an eslint rule. Always hard to agree and enforce on style/syntax when it's not possible to automatically check it.